NugetLightConstant.cs

```diff
@@ -28,7 +28,7 @@ public static class NuGetConstant
         /// <summary>
         /// Version of this provider
         /// </summary>
-        public static readonly string ProviderVersion = "2.8.5.209";
+        public static readonly string ProviderVersion = "3.0.0.0";
 
```

Comment:
we cannot use 3.0.0.0 because this version number is reserved for nuget 3.0 and we are following NuGet v2 protocol.